src/components/PostDetails/PostDetails.scss

```
@@ -7,6 +7,10 @@
     text-align: right;
   }
 
+  &__visiblelist {
+    display: none;
+  }
+
   &__list {
     list-style: none;
   }
```

According to BEM, this `&__visiblelist` [should actually be a modifier class] (https://mate-academy.github.io/fe-program/css/typical-bem-mistakes-en)of the `.PostDetails__list` element. Also, considering it has a `display: none` property, it should be called `invisible` or `hidden` :).

```suggestion
  &__list {
    list-style: none;

    &--hidden {
      display: none;
    }
  }
```